scripts/build_test.xml

```diff
@@ -102,15 +102,14 @@
 			<then>
 				<exec executable="cp" failonerror="true">
 					<arg value="-rf" />
-					<arg value="${ascii.workspace.dir}" />
+					<arg value="${ascii.workspace.dir}/TKG" />
 					<arg value="${ascii.test.dir}" />
 				</exec>
 			</then>
 			<else>
 				<copy todir="${ascii.test.dir}">
 					<fileset dir="${ascii.workspace.dir}">
 						<include name="TKG/**" />
-						<include name="*.mk" />
```

Comment:
Discussed with @smlambert and @llxia , we still need this for local run.
@psoujany Could you update to add this line back? Thanks.

Reply:
Thank you @LongyuZhang , updated as suggested.